index.js

```diff
@@ -2,13 +2,28 @@ const express = require("express");
 const port = process.env.PORT || 3000;
 const app = express();
 const addBookRouter = require('../server/routes/addBook')
-const volleyball = require('volleyball')
+const morgan = require('morgan')
 const cors = require('cors')
+const dotenv = require('dotenv')
+
+dotenv.config()
 
 app.use(cors());
 app.use(express.json())
 app.use(addBookRouter)
-app.use(volleyball)
+app.use(morgan('dev'))
+
+const mongoose = require("mongoose");
+mongoose.connect(process.env.DATABASE_URL, {
+  useNewUrlParser:true,
```

Comment:
Install a linter like `eslint` to catch formatting issues like the missing space here.